src/current/v24.3/ui-overview.md

```diff
@@ -16,7 +16,7 @@ Refer to: [Network Authorization for CockroachDB Cloud Clusters&mdash;DB Console
 
 ## Authentication
 
-The DB Console supports username/password login, as well single sign-on (SSO) ({{ site.data.products.dedicated }} and {{ site.data.products.core }} {{ site.data.products.enterprise }} clusters only).
+The DB Console supports username/password login and single sign-on (SSO) for {{ site.data.products.dedicated }} and {{ site.data.products.core }} {{ site.data.products.enterprise }} clusters.
```

Comment:
Same comment as earlier re: use of `core` variable. Also, `dedicated` needs to be replaced with presumably `advanced`? Someone else (@mdlinville ?) should verify.

```suggestion
The DB Console supports username/password login and single sign-on (SSO) for {{ site.data.products.dedicated }} and {{ site.data.products.enterprise }} clusters.
```

Reply:
I have applied your change to remove `core` as well as changed the `dedicated` to `advanced`!